File: Services/Dto/ServiceBudgetBench/ServiceBudgetDto.cs
using System;
using System.Collections.Generic;

namespace Services.Dto.ServiceBudgetBench

{
    public class ServiceBudgetDto
    {
        public int Id { get; set; }
        public int CustomerId { get; set; }
        public CustomerDto Customer { get; set; }
        public DateTime BudgetStartedIn { get; set; }
        public string Visually { get; set; }
        public string RemoteAccessData { get; set; }
        public string ClientProblems { get; set; }
        public DateTime BenchStartedIn { get; set; }
        public string Status { get; set; }
        public bool Authorized { get; set; }
        public List<SolutionPriceDto> SolutionsPrices { get; set; }
    }
}

File: Views/src/components/services-provision/dtos/service-budget-dto.ts
import { ClientDto } from "src/components/client/dto/client-dto";
import { SolutionPriceDto } from "./solution-price-dto";

export class ServiceBudgetDto {
  id: number;
  client: ClientDto;
  clientId: number;
  budgetStartedIn: Date;
  benchStartedIn: Date;
  clientProblems: string;
  status: string;
  visually: string;
  osMake: boolean;
  finished: boolean;
  solutionsPrices: SolutionPriceDto[]
}
